src/platforms/common/data-management/debug-files.mdx

```diff
@@ -211,6 +220,22 @@ that is not required to process minidumps. Most notably, inline functions are
 not declared, such that Sentry is not able to display inline frames in stack
 traces.
 
+### WASM
+
+</PlatformSection>
+
+For WebAssembly we support [DWARF in WASM containers](https://yurydelendik.github.io/webassembly-dwarf/).
+Note that we do not support source maps which are also a format used for WASM
+debugging but have shortcommings that make them impractical for a crash reporting
+tool like Sentry.
+
+Since WASM does not specify debug/build IDs yet we provide a separate tool to
+add build IDs and split files called [wasm-split](https://github.com/getsentry/symbolicator/tree/master/wasm-split)
+that can help you create a debug companion file ready for uploading to us
+while removing all debug information from the release binary.
```

Comment:
```suggestion
Since WASM does not specify debug/build IDs yet, we provide a separate tool to
add build IDs and split files called [wasm-split](https://github.com/getsentry/symbolicator/tree/master/wasm-split)
to help you create a debug companion file ready for uploading to Sentry
while removing all debug information from the release binary.
```